src/main/java/Deadline.java

```diff
@@ -0,0 +1,55 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Deadline class Represents an Deadline task that contains a description along with the date and
+ * time.
+ */
+public class Deadline extends Task {
+
+    protected String by;
+
+    /**
+     * Constructor that takes in the deadline description and the date/time
+     * @param description event description
+     * @param by date/time of event
+     */
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    /**
+     * Returns a modified string representation of the date/time
+     * @return modfied string representation of the date/time
```

Comment:
I think you forgot to add full stops here?😅 I noticed this for other Javadoc comments too